services/cell_test.go

```diff
@@ -47,4 +52,17 @@ func TestWriteCell(t *testing.T) {
 	styleIndex, err := file.Excel.GetCellStyle("test", "A2")
 	assert.Nil(t, err)
 	assert.Equal(t, styleIndex, 1, "Style index should be \"1\"")
+
+	cellFormula := &models.Cell{
+		Value:     "SUM(B2:B5)",
+		Style:     styles,
+		IsFormula: true,
+	}
+
+	err = file.writeCell("test", cellFormula, "A5")
+	assert.Nil(t, err)
+
+	value, err = file.Excel.GetCellFormula("test", "A5")
+	assert.Nil(t, err)
+	assert.Equal(t, value, "SUM(B2:B5)", "Value should be \"SUM(B2:B5)\"")
```

Comment:
This getCellValue should return the expected formula value or the formula?

Reply:
GetCellFormula should return the formula, not the calculated value